io.github.MakovWait.Godots.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Godots
+GenericName=Libre game engine version manager
+Comment=🚀Ultimate go-to hub for managing your Godot versions and projects! 🚀 
```

Comment:
I'm not sure how emoji are handled in general with desktop entries. but I'd recommend not using them.